dhis-2/dhis-web-api-test/src/test/java/org/hisp/dhis/webapi/controller/GistFieldsControllerTest.java

```diff
@@ -97,23 +97,13 @@ void testField_Remove_MinusSyntax()
         assertFalse( user.has( "surname" ) );
     }
 
-    @Test
-    void testField_Complex_SquareBracketsSyntax()
```

Comment:
We should add another test in a different domain here - we can do this in another PR